src/components/NoteList/NoteList.tsx

```diff
@@ -1,12 +1,33 @@
-import { FaTrash } from 'react-icons/fa';
-import type { TNote } from 'types';
+import { ContentState, convertToRaw, EditorState } from 'draft-js';
+import { useCallback } from 'react';
+import { FaEdit, FaTrash } from 'react-icons/fa';
+import { RiAddLargeFill } from 'react-icons/ri';
+import type { TNoteWithEditorState } from 'types';
 
+import { useAppDispatch } from '../../hooks/useAppDispatch';
+import { handleCreateNote } from '../../store/notes';
 import type { NoteListProps } from './types.ts';
 
 const NoteList = ({ notes, activeId, setActiveId, onDelete }: NoteListProps) => {
+  const dispatch = useAppDispatch();
+  const handleCreate = useCallback(() => {
+    const title = `Note ${Date.now()}`;
+    const contentState = ContentState.createFromText('');
+    const plainText = contentState.getPlainText();
+
+    dispatch(handleCreateNote(title, plainText));
+    console.log('use dispatch to create note');
+  }, [dispatch]);
+
```

Comment:
you already have https://github.com/aleksey-pru/notes_app/pull/5/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R36

you can remove this here